protractor-conf.js

```diff
@@ -20,7 +20,7 @@ exports.config = {
   capabilities: {
     browserName: 'chrome',
     chromeOptions: {
-      binary: './dist/win/devsuite-win32-x64/devsuite.exe'
+      binary: './dist/' + process.platform + '-' + process.arch +'/devsuite-win32-x64/devsuite.exe'
```

Comment:
For consistency's sake, I suggest replacing the 'devsuite-win32-x64' part as well.